EIPS/eip-1066.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 1066
+title: Status Codes
+author: Brooklyn Zelenka (@expede) <brooklyn@finhaven.com>, Tom Carchrae (@carchrae) <tom@finhaven.com>, Gleb Naumenko (@naumenkogs) <gleb@finhaven.com>
```

Comment:
I'm afraid the formatting doesn't presently support linking both emails and usernames - please pick one.

Reply:
Ah, okay. Will update!